test/testvar.cpp

```diff
@@ -539,8 +539,8 @@ TEST(Var, productOfSymbolAndSum)
 }
 
 TEST(Var, productOfConstantSumAndSum)
-    /* A sum of constants is treated as a constant when multiplied with a sum: (2 + sqrt(2))*(a + b)
-     * = (2 + sqrt(2))*a + (2 + sqrt(2))*b. */
+    /* A sum of constants is treated as a constant when multiplied with a sum: (2 + sqrt(2))*(a +
+    b)* = (2 + sqrt(2))*a + (2 + sqrt(2))*b. */
```

Comment:
please reformat the comment such that the linebreak is after the colon '[...] with a sum:\n    * (2 + sqrt(2))[...]'